csharp/src/Ice/ObjectAdapter.cs

```diff
@@ -132,6 +136,9 @@ public async Task ActivateAsync()
                     throw new InvalidOperationException($"object adapter {Name} already activated");
                 }
 
+                Interceptors.AddRange(interceptors);
```

Comment:
When you construct a communicator, you can specify dispatch interceptors in the constructor. Then you can add more interceptors _after_ these interceptors using plugins. So it seems more logical to me that interceptors specified in Activate are added after the existing interceptors, not before.